recipes/recastnavigation/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "20200511":
```

Comment:
The last release is from 2014, so you could ask to the author for a new release.

Reply:
Some people have already requested a new version https://github.com/recastnavigation/recastnavigation/issues/434

Reply:
Please prepend with `cci.`.
See https://github.com/conan-io/conan-center-index/pull/2437